paasta_tools/check_services_replication_tools.py

```diff
@@ -249,10 +272,26 @@ def get_mesos_tasks_and_slaves(
 
 
 def get_kubernetes_pods_and_nodes(
-    namespace: str,
+    namespace_prefix: str = None,
+    namespace: str = None,
+    additional_namespaces: List[str] = None,
```

Comment:
I think these should be marked as Optional.

(Or, for `additional_namespaces`, switch to `Sequence[str]` and switch the default value to an empty tuple)